docs/sdk/v3/guides/liquidity/01-minting-a-position.md

```diff
@@ -3,4 +3,90 @@ id: minting
+
+:::info
+
+For a briefer on the SDK and to learn more about how these guides connect to the examples repository, please visit our [background coverage](./01-background.md) page!
+
+:::
+
+In this example we will use  the `NonfungiblePositionManager` class to help us mint a liquidity position for the  **USDC - DAI** pair.
+The inputs are the **token in**, the **token out**,  the **amount in**, the **fee** and the **liquidity**.
+
+The guide will **cover**:
+1. Giving approval to the `NonfungiblePositionManager` contract to transfer our tokens.
+2. Fetching the Pool's constants and current state.
+3. Creating an instance of a `Position` class and using `addCallParameters` on our `NonfungiblePositionManager` to get the data for the transaction.
+
+At the end of the guide, given the inputs above with, we should be able to mint a liquidity position with the press of a button and view the position's id on the web application.
+
+## Example
+
+### Giving approval to the `NonfungiblePositionManager` contract to transfer our tokens
+
+To be able to provide liquidity to the Uniswap V3 protocol using our tokens, we need to mint a non-fungible token (NFT) which represents our position.
+
+The first step is to give approval to the protocol's `NonfungiblePositionManager` to transfer our tokens. 
+
+```js reference title="Approving our tokens for transferring" referenceLinkText="View on Github" customStyling
+https://github.com/Uniswap/examples/blob/6fba6da4d323804db56b3189ad1bbbaf18e6180f/v3-sdk/minting-position/src/example/Example.tsx#L113-L124
+```
+
+The logic to achieve that is wrapped in the `getTokenTransferApprovals` function. In short, since both **USDC** and **DAI** are ERC20 tokens, we setup a reference to their smart contracts and call the `approve` function:
+
+```js reference title="Setting up an ERC20 contract reference and approving" referenceLinkText="View on Github" customStyling
+https://github.com/Uniswap/examples/blob/6fba6da4d323804db56b3189ad1bbbaf18e6180f/v3-sdk/minting-position/src/libs/contracts.ts#L73-L78
+```
+
+### Fetching the Pool's constant and state to create an instance of a `Pool` class
+
+Having approved the transfer of our tokens, we now need to get data about the pool for which we will provide liquidity, in order to instantiate a Pool class:
+
+```js reference title="Fetching the Pool's constants and current state" referenceLinkText="View on Github" customStyling
+https://github.com/Uniswap/examples/blob/6fba6da4d323804db56b3189ad1bbbaf18e6180f/v3-sdk/minting-position/src/example/Example.tsx#L134-L135
+```
+
+:::info
+
+For a briefer on how to instantiate a reference to the Uniswap V3 Pool smart contract, and to retrieve the pool constants, please take a look at the [quoting guide](../quoting#setting-up-a-reference-to-the-pool-contract-and-getting-metadata-from-it)!
+
+:::
```

Comment:
I think we can assume developers have done this, but if not we can link a small bit of text IMO vs an entire info box. We should reserve these for important things otherwise users will learn to not pay attention to them

Reply:
e.g `similar to the [quoting example](link)`